doc/sphinx/conf.py

```diff
@@ -183,18 +183,27 @@ def setup(app):
 nitpicky = True
 
 nitpick_ignore = [ ('token', token) for token in [
-    'tactic',
-    # 142 occurrences currently sort of defined in the ltac chapter,
-    # but is it the right place?
-    'module',
-    'redexpr',
-    'modpath',
-    'dirpath',
     'collection',
+    'command',
+    'dirpath',
+    'modpath',
+    'module',
+    'red_expr',
+    'symbol',
+    'tactic',
+    'term_evar',
     'term_pattern',
     'term_pattern_string',
-    'command',
-    'symbol' ]]
+
+    #added for Gallina chapter
+    'arg',
+    'binders',
+    'cofix_body',
+    'evar_instance',
+    'match_item',
+    'mult_pattern',
+    'record_fields',
```

Comment:
Temporary OK (like `evar_term`). Please move in the alphabetical order like the res.